code/game/objects/structures/tank_dispenser.dm

```diff
@@ -24,19 +24,26 @@
 
 /obj/structure/dispenser/New()
 	. = ..()
+	while(oxygentanks>0)
```

Comment:
They have a value of oxygentanks = 0 (you can see this at the top of this file)

And of course I tested it just to be sure, and to check icon states and stuff